website/docs/developer/research/research-orca-delete-functionality.mdx

```diff
@@ -0,0 +1,179 @@
+---
+id: research-orca-delete-functionality
+title: Research Notes on ORCA delete functionality.
+description: Research Notes on implementing delete functionality for ORCA.
+---
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Delete functionality use cases
+
+Some use cases for delete functionality are as follows:
+
+- As a DAAC, I would like to remove data from ORCA that is invalid or expired in order to maintain a clean data archive.
+    This can be achieved by a two-step process- an initial soft delete initiated by user, and a physical delete to permanently delete the data from S3 and database.
+    It is assumed that the process is not automated and must be initiated by the user with an API call for both steps.
+
+- As a DAAC, I would like to remove data from ORCA only if it has been in glacier for more than 90 days from initial ingest to keep from incurring additional cost for early removal.
+
+    This can be implemented by using soft delete. We can either flag the data in the database with a new column named `Pending Deletion` or we can create a separate table that contains those data marked for deletion. Then after 90 days, we can permanently delete from the DB as well as the glacier bucket.
+    If the data is stored in standard it would be 90 days but for deep glacier, it would be 180 days. If the time the data stays in ORCA is less than the initial (90/180) days from ingest, any kind of delete operation will not be allowed. Some business logic should be added to the code to execute this rule.
+
+
+
+- As a DAAC, I would like to configure the amount of time data remains in ORCA after being signaled to be deleted.	
+
+    This should be configurable by each DAAC that is using ORCA. The number of days it would stay in ORCA would be based off of the soft delete date. Once the data stays in ORCA for greater than 90/180 days based on the glacier storage type used, that data would be eligible for a physical delete.
+    The data should stay in ORCA for x number of days for safety reasons. The configuration should be done using an environment variable during deployment. It is assumed that the configuration cannot be overridden except with a new deployment.
+    This is applicable for physical deletes. Note that it should also prevent deletion of certain data such as Level 0. 
```

Comment:
Fix this a little bit by combining with paragraph below to maybe something like 
```
Additionally, ORCA should prevent deletion of any kind from certain data like Level 0 or base data sets that 
are static. Some ways to implement this would be to have a configuration parameter we capture that says 
we are not able to delete from this collection or use an S3 bucket having bucket policies that prevents any 
deletes from happening.
```

Reply:
updated the wording